File: app/src/main/java/com/artmurka/artmurkaapp/android/views/fragments/interfaces/IBasketFragment.java
package com.artmurka.artmurkaapp.android.views.fragments.interfaces;

import com.artmurka.artmurkaapp.model.pojo.itemlist.itembasket.Item;

import java.util.List;


public interface IBasketFragment {
    void showError(String error);
    void showItemsInBasket(List<Item> items);
    void makeMessageInvisible(boolean b);
    void showPrice(String price);
}

File: app/src/main/java/com/artmurka/artmurkaapp/android/views/fragments/interfaces/IBasketFragment.kt
package com.artmurka.artmurkaapp.android.views.fragments.interfaces

import com.artmurka.artmurkaapp.data.model.pojo.itemlist.itembasket.Item
import com.artmurka.artmurkaapp.presenter.PresenterView


interface IBasketFragment : PresenterView {
    fun showError(error: String)
    fun showItemsInBasket(items: List<Item>)
    fun makeMessageInvisible(b: Boolean)
    fun showPrice(price: String)
}
